changelogs/fragments/22-jira.yaml

```
@@ -0,0 +1,8 @@
+---
+minor_changes:
+- Added serarch function with support for Jira JQL
+- Added update function which can update Jira Selects etc.
+
+bugfixes:
+- Priting full error message from jira server
+- Issue 109: Transsistion issue not working 
```

```suggestion
- jira - transsistion issue not working (https://link/to/the/issue)
```
if issue doesn't exist, put the link to this PR there. And the same for the other lines in ```bugfixes``` / ```minor_changes``` too.

(i didn't know that there are several minor changes/bugfixes. For the future, it's worth using one PR for one change/fix not to mix a lot of things in one place - at least, much easier to review)

---

Transsistion -> transition

---

Priting -> printing
Don't rush:)